src/components/landing/LandingScreen/index.js

```diff
@@ -0,0 +1,13 @@
+import VideoContainer from '../VideoContainer';
+import './styles.scss';
+
+const LandingScreen = ({ children }) => {
```

Comment:
what do you think about using prop types? because children as prop is too generic. I strongly recommend for you to take a look on it. Feel free to ask me for help if needed.

Reply:
Children is a special word for components that are inside of the component, like
`<LandingScreen>__<div />__ </LandingScreen>`
Should we try to type it anyways?